flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/MiniDispatcherTest.java

```diff
@@ -369,7 +377,9 @@ private MiniDispatcher createMiniDispatcher(
                         // JobManagerRunnerRegistry needs to be added explicitly
                         // because cleaning it will trigger the closeAsync latch
                         // provided by TestingJobManagerRunner
-                        .withLocallyCleanableResource(jobManagerRunnerRegistry)
+                        .withLocallyCleanableResource(
```

Comment:
Here we are inconsistent. The `TestingResourceCleanerFactory` has a dedicated `withLocallyCleanableInMainThreadResource` method for this purpose. Or am I missing something? :thinking:

Reply:
Correct, thanks!